server/config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+3Qf3aVRtI+iL4j6eHD+dwqnVI8z02xxOJesRcBc1ORSQyTJM3UzOZ351EI30VDbNsNNqoenZprC07uoC3EKlOqECyarkfA1HlT0RsU1Y7h2Dma9hYx5flp9NqGGXnsI354qecVjXlnOWe2HPRWz1sl08CbIgLVH+9N2wQYaQbLZCwtbvyUCrBvvWGIzpBwroOM/kig5Eqt/5WtMm18Bdi/hho7LnC7qYRbG3l3gtbM2QnqRmVPmKFo039OzTIU4kI4YwKgT8SvXQexslu6J5AyV1/TsG/1IAjckaAlTw+MDtgWvYEeUz7tZQO1ymlHRPgzhL0yNT5/CebdGHNJXGz/FRfzZB3a/ZTvscVzUzs7z7vlgoGdKe8KUrARiCUO7vHbDp5NVzIZbmzgXgLb9zt3gnUaiKETJKEobOOTryMGrXusCBtikqJVtKeM5qEPgAsJ3BiCEekXgscUz4qQgzsXYdlpyDYcz5cddd4Lf5GBibzBmAOXIlpH6+pz++BDWQKWrSH1hBumNmODPgG4YZ1Peqd2cDCID41LR8mrv1NBIZo8iJcnmh4nK49dNIKJuISSvBCrhFQ+FQM7Hu1thBJlrFiw4nhTGIC4ugRTO4onVLlw==--pNrwgyOIPjC0Je0k--l+LRWfP6rrN7zymkVsYoyQ==
```

Comment:
this file should not be added as your credentials looks different from setup of other developers